packages/static-build/src/utils/gatsby.ts

```diff
@@ -92,6 +92,8 @@ async function updateGatsbyConfig(dir: string, plugins: string[]) {
   }
 }
 
+const GENERATED_FILE_COMMENT = `// This file was generated by @vercel/static-build`;
```

Comment:
I would suggest something more generic like:

```
This file was auto-generated by Vercel
```

Not a huge preference though.